h/templates/panels/navbar.html.jinja2

```diff
@@ -13,6 +13,20 @@
             id="search-bar"
             action="{{ search_url }}">
         {{ svg_icon('search', 'search-bar__icon') }}
+        <div class="search-bar__special-lozenges">
```

Comment:
I would suggest pulling this out into a helper macro to clearly indicate that it is a separate component, and also adding a comment to the effect that it is the same markup as generated by `LozengeController` on the client.